docs/release-notes/2.24.0.md

```diff
@@ -0,0 +1,53 @@
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8027](https://jira.percona.com/browse/PMM-8027): Integrated Alerting: New notification channel added: Webhooks
+- [PMM-8301](https://jira.percona.com/browse/PMM-8301): Add data collection support and visualization for newly added dimensions in pg_stat_monitor such as Application Name,Top Query, Plan in Query Analytics
+- [PMM-8588](https://jira.percona.com/browse/PMM-8588): PostgreSQL Histograms added to QAN when using pg_stat_monitor extension
+- [PMM-8632](https://jira.percona.com/browse/PMM-8632): New Filter: "Command Type" allows filtering queries based on type (SELECT, INSERT, UPDATE, DELETE, n/a) when pg_stat_monitor extension enabled
+
+## Improvements
+
+- [PMM-8803](https://jira.percona.com/browse/PMM-8803): Backup Management: Improved error messages to indicate incompatible versions of software
+- [PMM-8636](https://jira.percona.com/browse/PMM-8636): Integrated Alerting: Additional context to alerts to better convey issue detected
+- [PMM-8644](https://jira.percona.com/browse/PMM-8644): Integrated Alerting: API should allow textual TLS configs for webhooks
+- [PMM-8122](https://jira.percona.com/browse/PMM-8122): Integrated Alerting: UI does not indicate a port is needed in configuration for SMTP communication channel
+- [PMM-8484](https://jira.percona.com/browse/PMM-8484): Added support for PostgreSQL 14
+- [PMM-7297](https://jira.percona.com/browse/PMM-7297): update vertamedia-clickhouse-datasource plugin
+
+## Bugs Fixed
+
+- [PMM-8975](https://jira.percona.com/browse/PMM-8975): Scheduled backups - cron expression incorrectly parsed
+- [PMM-8541](https://jira.percona.com/browse/PMM-8541): Navigating through PMM Settings link at Failed security checks panel takes more than 30 seconds
+- [PMM-8387](https://jira.percona.com/browse/PMM-8387): MySQL InnoDB Details dashboard is not in the left menu
+- [PMM-8858](https://jira.percona.com/browse/PMM-8858): Dashboards: No Host uptime on Homepage for RDS instances 
+- [PMM-8611](https://jira.percona.com/browse/PMM-8611): Agents state shows DOWN if the respective metric is unknown
+- [PMM-8393](https://jira.percona.com/browse/PMM-8393): Alert rules not executed after upgrading PMM Server running as Docker container
+- [PMM-8058](https://jira.percona.com/browse/PMM-8058): Integrated Alerting: Firing alerts disappear after docker restart 
+- [PMM-8089](https://jira.percona.com/browse/PMM-8089): When `mapped` or `mapped_with_journal` returns no values, then exporter fails to generate any metrics
+- [PMM-9100](https://jira.percona.com/browse/PMM-9100): SHOW BINARY LOGS not parsed properly for MySQL8
+- [PMM-8633](https://jira.percona.com/browse/PMM-8633): Incorrect use of state_code in monitoring with pg_stat monitor. Unfinished queries are included in statistics
```

Comment:
```suggestion
- [PMM-8633](https://jira.percona.com/browse/PMM-8633): Unfinished queries are included in Query Analytics for PostgreSQL with pg_stat_monitor usage because of incorrect use of state_code. 
```